packages/core-data/src/components/SearchListItem.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import React from 'react';
+import Icon from './Icon';
+import { type Attribute } from '../types/SearchList';
+
+type Props = {
+  /**
+   * Title of the record
+   */
+  title: string,
+  /**
+   * Attributes to display (only the first three are shown)
+   */
+  attributes: Attribute[],
+  /**
+   * Search item object
+   */
+  item: any
+};
+
+const ListItem = (props: Props) => (
+  <li className='py-3 px-6'>
+    <p className='font-bold text-neutral-800'>{props.title}</p>
+    {props.attributes.length > 0 && (
+      <ul className='list-none'>
+        {props.attributes.slice(0, 3).map((att) => (
+          <li
+            className='text-sm text-neutral-800 flex gap-2 items-center list-none pl-5 pt-1'
+            key={att.name}
+          >
+            <Icon
```

Comment:
Should we make it possible to omit the icon entirely?

Reply:
I don't have a problem with that, but this is what the designs say:

<img width="260" alt="Screenshot 2025-02-11 at 1 55 48 PM" src="https://github.com/user-attachments/assets/d7392226-305a-4e63-a2db-706c2f201fa6" />